packages/Webkul/BookingProduct/src/Listeners/Order.php

```diff
@@ -32,5 +33,15 @@ public function __construct(BookingRepository $bookingRepository)
     public function afterPlaceOrder($order)
     {
         $this->bookingRepository->create(['order' => $order]);
+
+        foreach($order->items as $item) {
+            if($item->type !== 'booking') {
```

Comment:
Hi @Tami93,
I think, your code is a little bit more readable if you change it as follows:
```php
if ($item->type === 'booking') {
    $eventTicketId = $item->additional
}
```